packages/contracts-bedrock/scripts/libraries/Config.sol

```diff
@@ -129,6 +129,26 @@ library Config {
         env_ = vm.envString("ETHERSCAN_API_KEY");
     }
 
+    /// @notice Returns the expected superchain config address for OPCM verification.
+    function expectedSuperchainConfig() internal view returns (address env_) {
+        env_ = vm.envOr("EXPECTED_SUPERCHAIN_CONFIG", address(0));
+    }
+
+    /// @notice Returns the expected protocol versions address for OPCM verification.
+    function expectedProtocolVersions() internal view returns (address env_) {
+        env_ = vm.envOr("EXPECTED_PROTOCOL_VERSIONS", address(0));
+    }
+
+    /// @notice Returns the expected superchain proxy admin address for OPCM verification.
+    function expectedSuperchainProxyAdmin() internal view returns (address env_) {
+        env_ = vm.envOr("EXPECTED_SUPERCHAIN_PROXY_ADMIN", address(0));
+    }
+
+    /// @notice Returns the expected upgrade controller address for OPCM verification.
+    function expectedUpgradeController() internal view returns (address env_) {
+        env_ = vm.envOr("EXPECTED_UPGRADE_CONTROLLER", address(0));
+    }
```

Comment:
let's stick to the approach currently used of disabling semgrep and reading env vars:

https://github.com/ethereum-optimism/optimism/blob/9b90b39bd3e10ac308bb888bf134ebf599ac09f3/packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol#L128